zamba/models/config.py

```diff
@@ -93,8 +93,8 @@ def check_files_exist_and_load(
     Returns:
         pd.DataFrame: DataFrame with valid and loadable videos.
     """
-    # update filepath column to prepend data_dir if filepath column is not an absolute path
-    data_dir = Path(data_dir).resolve()
+    # update filepath column to prepend data_dir
+    # NOTE: A previous version of this resolved all paths to absolute, but that breaks caching.
```

Comment:
[nit] do you think we need this note here? the only reason this breaks caching is because we provided a previously cached directory (which is not a supported user use case). the absolute path works fine if you let zamba handle the caching on the first epoch. i'm inclined to remove this comment since i think it contains inside baseball and doesn't help us in the future

Reply:
Agreed